code/modules/projectiles/ammo_datums.dm

```diff
@@ -1492,7 +1495,18 @@ datum/ammo/bullet/revolver/tp44
 	sundering = 65
 
 /datum/ammo/rocket/atgun_shell/apcr/drop_nade(turf/T)
-	explosion(T, 0, 0, 1, 0)
+	explosion(T, 0, 0, 0, 1)
```

Comment:
Not a fan, remove that for now and nerf again if needed

Reply:
damage buffed to 200 per discord